cpp-ch/local-engine/Parser/aggregate_function_parser/CommonAggregateFunctionParser.cpp

```diff
@@ -62,4 +62,6 @@ REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(Ntile, ntile, ntile)
 REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(PercentRank, percent_rank, percent_rank)
 REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(CumeDist, cume_dist, cume_dist)
 
+// Only for ut
+REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(CustomSum, custom_sum, sum)
```

Comment:
Moving `REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER` into header  and then moving `REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(CustomSum, custom_sum, sum)` into seperated cpp.

Refer to [cpp-ch/local-engine/Parser/example_udf/myAdd.cpp](https://github.com/oap-project/gluten/blob/main/cpp-ch/local-engine/Parser/example_udf/myAdd.cpp) and #1589

Reply:
Done